skimage/restoration/_unwrap_2d.pyx

```diff
@@ -3,21 +3,23 @@
 # cython: nonecheck=False
 # cython: wraparound=False
 
+cimport numpy as cnp
+
 
 cdef extern from "unwrap_2d_ljmu.h":
     void unwrap2D(
-            double *wrapped_image,
-            double *UnwrappedImage,
+            cnp.float64_t *wrapped_image,
+            cnp.float64_t *UnwrappedImage,
```

Comment:
I wasn't sure whether to keep "double" for the extern function
definition e.g. in _unwrap_2d.pyx and _unwrap_3d.pyx but I opted for the
explicit and consistent option.